application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/wolframalpha/WolframAlphaCommand.java

```diff
@@ -0,0 +1,291 @@
+import org.togetherjava.tjbot.commands.SlashCommandAdapter;
+import org.togetherjava.tjbot.commands.SlashCommandVisibility;
+import org.togetherjava.tjbot.commands.utils.WolfCommandUtils;
+import org.togetherjava.tjbot.config.Config;
+
+import javax.imageio.ImageIO;
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.Graphics;
+import java.awt.font.TextAttribute;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.net.URL;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+public final class WolframAlphaCommand extends SlashCommandAdapter {
+    public static final Logger logger = LoggerFactory.getLogger(WolframAlphaCommand.class);
+    private static final int HTTP_STATUS_CODE_OK = 200;
+    private static final XmlMapper XML = new XmlMapper();
+    private static final String QUERY_OPTION = "query";
+    /**
+     * WolframAlpha API endpoint to connect to.
+     *
+     * @see <a href=
+     * "https://products.wolframalpha.com/docs/WolframAlpha-API-Reference.pdf">WolframAlpha API
+     * Reference</a>.
+     */
+    private static final String API_ENDPOINT = "http://api.wolframalpha.com/v2/query";
+    private static final int MAX_IMAGE_HEIGHT_PX = 300;
+    /**
+     * WolframAlpha text Color
+     */
+    private static final Color WOLFRAM_ALPHA_TEXT_COLOR = Color.decode("#3C3C3C");
+    /**
+     * WolframAlpha Font
+     */
+    private static final Font WOLFRAM_ALPHA_FONT = new Font("Times", Font.PLAIN, 15).deriveFont(
+            Map.of(TextAttribute.UNDERLINE, TextAttribute.UNDERLINE_ON));
+    private final HttpClient client = HttpClient.newHttpClient();
+
+    public WolframAlphaCommand() {
+        super("wolf", "Renders mathematical queries using WolframAlpha",
+                SlashCommandVisibility.GUILD);
+        getData().addOption(OptionType.STRING, QUERY_OPTION, "the query to send to WolframAlpha",
+                true);
+    }
+
+    @Override public void onSlashCommand(@NotNull SlashCommandEvent event) {
+
+        // The processing takes some time
+        event.deferReply().queue();
+        String query = Objects.requireNonNull(event.getOption(QUERY_OPTION)).getAsString();
+        HttpRequest request = sendQuery(query);
+        Optional<HttpResponse<String>> optResponse = getResponse(event, request);
+        if (optResponse.isEmpty())
+            return;
+        HttpResponse<String> response = optResponse.get();
+        Optional<QueryResult> optResult = parseQuery(response, event);
+        if (optResult.isEmpty())
+            return;
+        QueryResult result = optResult.get();
+        Optional<WebhookMessageUpdateAction<Message>> optAction = createResult(result, event);
+        optAction.ifPresent(RestAction::queue);
```

Comment:
```suggestion
        event.deferReply().queue();
        
        String query = Objects.requireNonNull(event.getOption(QUERY_OPTION)).getAsString();
        
        HttpRequest request = sendQuery(query);
        Optional<HttpResponse<String>> optResponse = getResponse(event, request);
        
        if (optResponse.isEmpty()) {
            return;
        }
        
        HttpResponse<String> response = optResponse.get();
        Optional<QueryResult> optResult = parseQuery(response, event);
        
        if (optResult.isEmpty()) {
            return;
        }
        
        QueryResult result = optResult.get();
        Optional<WebhookMessageUpdateAction<Message>> optAction = createResult(result, event);
        optAction.ifPresent(RestAction::queue);
```

Imo you should add more empty lines for readability, and use brackets with the ifs for consistency reasoning.
Find it myself a lot harder to read without if it's inside the code.